CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+Once you submit your PR, a team member will review your proposal. We may ask questions or request for additional information.
+- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch.
+- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
+- If you run into any merge issues, checkout this [git tutorial](https://lab.github.com/githubtraining/managing-merge-conflicts) to help you resolve merge conflicts and other issues.
+- Please make sure that all tests are passing, github pages renders nicely, and code coverage are are not lower than before your contribution. You see the different github action workflows by clicking the "Action" tab in the GitHub repository.
+
+
+### Enforced style guide using pre-commit hooks
+
+We want to have a consistent style on all the contributions to the repository. The way we enforce this is through pre-commit hooks and contributors are encouraged to install the pre-commit hooks locally when developing. You can install the pre commit hooks by first install `pre-commit`
+```
+python3 -m pip install pre-commit
+```
+and then install the pre-commit hooks using the command
+```
+pre-commit install
+```
+at the root of the repository. This will install all the hooks listed in the file called `.pre-commit-config.yaml` in the root of the repository.
+
+Every time you make a commit to the repository a set of tests will run to make sure that the changes you made are following the style guide. Usually, the hooks will autoformat your code so that you only need to do a `git add` again and then redo the `git commit`.
+
+Note that when you make a push to the repo, the pre-commit hooks will be run on all the files in the repository. You can also run the pre-commit hooks on all the files using the command
+```
+pre-commit run --all
+```
+To learn more about pre-commit you can check out https://pre-commit.com
+
+## Test suite
+For every new feature of bugfix you should also make sure to not lower the code coverage for the test suite. This means that if you for example add a new function then you should also make sure that the function is properly tested (at a minimum it should be covered by the test suite).
+
+To run the test suite, please install the package with the optional dependencies `test`, i.e
+```
+python3 -m pip install -e ".[test]"
```

Comment:
```suggestion
python3 -m pip install -e ".[all]"
```
Hvis vi vil sørge for at alle tester med varierende dependencies skal støttes, eventuelt legge inn `pytest.mark.skip`.